aea/skills/tasks.py

```diff
@@ -116,6 +116,7 @@ def init_worker() -> None:
     :return: None
     """
     signal.signal(signal.SIGINT, signal.SIG_IGN)
+    signal.signal(signal.CTRL_C_EVENT, signal.SIG_IGN)
```

Comment:
why not use your new helper method?

Reply:
it is not "send_signal". This just ignores the `CTRL_C_EVENT` signal on Windows.